{{ cookiecutter.project_name }}/template.yaml

```diff
@@ -8,23 +8,33 @@ Description: >
 # More info about Globals: https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst
 Globals:
     Function:
-        Timeout: 5
+        Timeout: 30
```

Comment:
```suggestion
        Timeout: 5
```

30 is a high timeout for a hello world, and might give the impression is a good practice. Please leave at 5s